src/sidebar/components/group-list.js

```diff
@@ -26,6 +40,18 @@ function GroupListController($window, analytics, groups, settings, serviceUrl) {
     }
   };
 
+  this.orgName = function (groupId) {
+    const group = this.groups.get(groupId);
+    return group && group.organization && group.organization.name;
```

Comment:
I would suggest we just assume that `group.organization` is defined and change the code if we ever revise that decision, since otherwise we may miss a bug if it is (incorrectly) missing.

Reply:
If the `group` is not expanded, `group.organization` won't be an object. Would you still like me to make this change?

Reply:
Ah true. Ignore this then.